.github/dependabot.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 5
+    ignore:
+    - dependency-name: "k8s.io/*"
+      update-types: ["version-update:semver-major", "version-update:semver-minor"] # ignore all except for patch updates
+    - dependency-name: "github.com/vmware/go-ipfix"
+    - dependency-name: "github.com/TomCodeLV/OVSDB-golang-lib"
+    - dependency-name: "github.com/contiv/libOpenflow"
+    - dependency-name: "github.com/ofnet/libOpenflow"
```

Comment:
I didn't find this "github.com/ofnet/libOpenflow" in go.mod, I am wondering what's the criteria for the dependency to be added here?

Reply:
This was a typo. But I removed these 2 dependencies from the list now that we are using our own forks in antrea-io. These dependencies are the ones for which we don't want automatic updates or for which we want restricted automatic updates. Often times this is because updating the dependency requires a lot of changes in Antrea (e.g. using a different minor version of the K8s module). Or if a module doesn't respect Go semantic versioning. This list may change over time but this is a good place to start. I included information about this in the PR description and in the original issue.

Reply:
thanks for the explanation, I got the idea from PR description, just confused by the typo 😄